3 - PB/Documentazione esterna/Specifica Tecnica/Specifica Tecnica.typ

```diff
@@ -73,6 +73,84 @@ Il prodotto offre le seguenti funzionalità principali:
 
 == Database
 
+In questa sezione viene presentato lo schema della base di dati realizzata con PostgreSQL.
+
+Esso è cosi composto:
+
+#figure(
+  image("ER.png", width: 100%),
+  caption: [
+    Schema ER del Database.
+  ],
+)
+
+
+=== Entità
+
+Il database è composto da 6 entità:
+
+- *Product*: rappresenta un prodotto presente all'interno del magazzino. Composto da:
+
+    - ID: identificativo univoco e seriale di un prodotto;
+    - Name: nome del prodotto;
+    - Weight: peso del prodotto;
+    - Width: larghezza del prodotto;
+    - Length: lunghezza del prodotto;
+    - Height: altezza del prodotto.
+
+- *Category*: rappresenta la categoria di appartenenza dei prodotti. Composto da:
```

Comment:
riformulerei in:
rappresenta la categoria merceologica di appartenenza del prodotto